src/Components/Books/Books.module.css

```diff
@@ -1,4 +1,273 @@
-* {
-  margin: 0;
-  padding: 0;
+@import url('https://fonts.googleapis.com/css2?family=Dangrek&family=Roboto+Slab:wght@200;300;400;500;600;700;800&display=swap');
+
+body {
+  background-color: #f4f5f9;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  padding: 0 6rem;
+}
+
+.Bookcontainer {
+  display: flex;
+  height: 10.625rem;
+  border-radius: 4px;
+  background-color: #fff;
+  justify-content: space-between;
+  align-items: center;
+  width: 80%;
+  margin: 12px auto;
+}
+
+.list {
+  height: 33rem;
+  overflow-y: scroll;
+  margin-top: 40px;
+}
+
+/* width */
+.list::-webkit-scrollbar {
+  width: 10px;
+}
+
+/* Track */
+.list::-webkit-scrollbar-track {
+  border-radius: 15px;
+}
+
+/* Handle */
+.list::-webkit-scrollbar-thumb {
+  background: #0290ffba;
+  border-radius: 15px;
+}
+
+/* Handle on hover */
+.list::-webkit-scrollbar-thumb:hover {
+  background: #0290ff;
+}
+
+.details {
+  flex-grow: 2;
+  padding-left: 1.688rem;
+  width: 25%;
+}
+
+.details h3 {
+  font-size: 0.875rem;
+  font-weight: bold;
+  font-stretch: normal;
+  font-style: normal;
+  line-height: normal;
```

Comment:
- [ ] Kindly give the category text the right font as seen in the original design 🙏 

Original Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/201356852-3a72cc60-b5ab-452e-80ed-cccb114331d4.png)  |  ![image](https://user-images.githubusercontent.com/74814780/201356816-e5cda8eb-739f-4585-8675-fe486261a111.png)